src/test/java/spoon/test/query_function/AllLocalVars.java

```diff
@@ -0,0 +1,263 @@
+package spoon.test.query_function;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.function.Consumer;
+
+/*
+ * The values of the fields must be a sequence starting from 1

```

Comment:
It is the class which is used to build spoon model, which is then used for searching for variable declarations and then back their variable definitions. It belongs to package ...testclass...

new AllLocalVars(), runs all the code of this class and checks that all assertions are correct - that all variable references really refers the expected variable declaration - that test model input is valid.